apps/frontend/src/api/__tests__/util.test.tsx

```diff
@@ -24,17 +24,6 @@ describe('useFetchOptions', () => {
     <authTestUtils.LoggedIn user={undefined}>{children}</authTestUtils.LoggedIn>
   );
 
-  it('sets the default headers', async () => {
```

Comment:
Might still want to leave it in since I don't assume the test breaks now? I don't really trust `use-http` not to break this logic in an update